webhook_test.go

```diff
@@ -181,7 +214,8 @@ func buildHookecho(t *testing.T) (binPath string, cleanupFn func()) {
 		binPath += ".exe"
 	}
 
-	cmd := exec.Command("go", "build", "-o", binPath, "test/hookecho.go")
+	gobin := filepath.Join(runtime.GOROOT(), "bin", "go")
+	cmd := exec.Command(gobin, "build", "-o", binPath, "test/hookecho.go")
```

Comment:
I have several different go versions installed on my development machine, so I wanted to be sure the `go` that builds the test code was the same `go` that runs the tests.